### ISSUE TITLE
Add LABELs to Dockerfiles

### ISSUE THREAD

**Issue by jhale** [FEniCS/dolfinx#2408]
GitHub Container Registry supports Markdown descriptive labels that appear as formatted text underneath an image. Could be used to provide examples of use that are currently hidden away in the `Dockerfile`.

```LABEL org.opencontainers.image.description DESCRIPTION```

**Comment by garth-wells** [comment]
@jhale can this be closed?

**Comment by jhale** [comment]
Yes, what I wanted to do, set e.g. git commit labels, is not possible.
